src/camera.js

```diff
@@ -332,6 +339,56 @@ vgl.camera = function () {
 
   ////////////////////////////////////////////////////////////////////////////
   /**
+   * Get parallel projection extents parameters
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.parallelExtents = function () {
+    return m_parallelExtents;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Set parallel projection extents parameters
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.setParallelExtents = function (width, height, zoom, tilesize) {
+    var extents = [width, height, zoom, tilesize], i, mod = false;
+    for (i = 0; i < extents.length; i += 1) {
+      if (extents[i] !== undefined && extents[i] !== m_parallelExtents[i]) {
+        m_parallelExtents[i] = extents[i];
+        mod = true;
+      }
+    }
+    width = m_parallelExtents[0];
+    height = m_parallelExtents[1];
+    zoom = m_parallelExtents[2];
+    tilesize = m_parallelExtents[3];
+    if (mod && width && height && zoom !== undefined && tilesize) {
+      m_unitsPerPixel = this.unitsPerPixel(zoom, tilesize);
+      m_right = m_unitsPerPixel * width / 2;
+      m_left = -m_right;
+      m_top = m_unitsPerPixel * height / 2;
+      m_bottom = -m_top;
+      this.modified();
+    }
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Compute the units per pixel.
+   *
+   * @param zoom: tile zoom level.
+   * @param tilesize: number of pixels per tile (defaults to 256).
+   * @returns: unitsPerPixel.
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.unitsPerPixel = function (zoom, tilesize) {
+    tilesize = tilesize || 256;
+    return 360.0 * Math.pow(2, -zoom) / tilesize;
```

Comment:
I'm not particularly happy with my choice of the `setParallelExtents` method, but I needed to know the zoom level and size of the screen.  I made it so you could also set the tile size, in case that was relevant.  I'm open to suggestions on improving this.